README.md

```diff
@@ -5,10 +5,10 @@
 `@imgix/js-core` is a JavaScript library for generating image URLs with [imgix](https://www.imgix.com/) that can be used in browser or server-side settings.
 
 [![NPM Version](https://img.shields.io/npm/v/imgix-core-js.svg)](https://www.npmjs.com/package/@imgix/js-core)
```

Comment:
Should be `[![NPM Version](https://img.shields.io/npm/v/@imgix/js-core.svg)](https://www.npmjs.com/package/@imgix/js-core)`